### DISCUSSION TITLE
Selecting ZLS version for zig?

### DISCUSSION THREAD

**Discussion by jared-cone** [zed-industries/zed#18794] [Feature Requests]
Is there a setting to specify which version of ZLS to use for zig? Zed shows a popup saying I'm using zig 0.11.0 but zls 0.13.0 (I have to use zig 0.11.0)

**Comment by omarjatoi** [comment]
I believe Zed downloads the latest ZLS from GitHub releases automatically. If you'd like to override the version, I believe you can install 0.11.0 locally and then configure Zed to look for the ZLS binary at a given path. I found this comment which includes the full instructions: https://github.com/zed-industries/zed/issues/11162#issuecomment-2266039681:

```json
{
  "lsp": {
    "zls": {
      "binary": {
        "path": "/path/to/home/.local/bin/zls"
      }
    }
  }
}
```

**Reply by jared-cone** [reply]
Yep that's got it thanks!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
